tests/rust/re_integration_test/tests/basic_tests.rs

```diff
@@ -0,0 +1,35 @@
+use re_integration_test::HarnessExt as _;
+use re_sdk::TimePoint;
+use re_sdk::log::RowId;
+use re_view_text_document::TextDocumentView;
+use re_viewer::external::re_types;
+use re_viewer::external::re_viewer_context::ViewClass as _;
+use re_viewer::viewer_test_utils;
+use re_viewport_blueprint::ViewBlueprint;
+
+#[tokio::test(flavor = "multi_thread")]
+pub async fn test_single_text_document() {
+    let mut harness = viewer_test_utils::viewer_harness();
+    harness.init_recording();
+    harness.toggle_selection_panel();
```

Comment:
See this is the problem with non-idempotency. Is this opening or closing the selection panel? 🤷